src/lfs.h

```diff
@@ -5,27 +5,32 @@
 
 /* Define 'chdir' for systems that do not implement it */
 #ifdef NO_CHDIR
-#define chdir(p)	(-1)
-#define chdir_error	"Function 'chdir' not provided by system"
+  #define chdir(p)	(-1)
+  #define chdir_error	"Function 'chdir' not provided by system"
 #else
-#define chdir_error	strerror(errno)
-
+  #define chdir_error	strerror(errno)
 #endif
 
 #ifdef _WIN32
-#define chdir(p) (_chdir(p))
-#define getcwd(d, s) (_getcwd(d, s))
-#define rmdir(p) (_rmdir(p))
-#ifndef fileno
-#define fileno(f) (_fileno(f))
-#endif
+  #define chdir(p) (_chdir(p))
+  #define getcwd(d, s) (_getcwd(d, s))
+  #define rmdir(p) (_rmdir(p))
+  
+  #define lfs_export __declspec (dllexport)
```

Comment:
it might be a matter of taste but... could you name it `LFS_EXPORT` rather than `lfs_export`? I think it's more usual for this kind of macro.